packages/wallets/jest.config.js

```diff
@@ -10,4 +10,5 @@ module.exports = {
         '\\.(ts|tsx)?$': '<rootDir>/jestTransformer.js',
         '^.+\\.jsx?$': 'babel-jest',
     },
+    transformIgnorePatterns: ['/node_modules/(?!(@deriv-com/ui)).+\\.js$'],
```

Comment:
what does this do?

Reply:
The test was failing due to  @deriv-com/ui. I noticed that another application had included it as well.